packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -1,25 +1,33 @@
-import PropTypes from 'prop-types';
 import React from 'react';
-import { withRouter } from 'react-router-dom';
-import { loginUrl, routes, PlatformContext } from '@deriv/shared';
+import { withRouter, useHistory } from 'react-router-dom';
+import { loginUrl, routes, redirectToLogin, PlatformContext } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
 import { getLanguage } from '@deriv/translations';
-import { connect } from 'Stores/connect';
 import { WS } from 'Services';
 
-const Redirect = ({
-    history,
-    currency,
-    setVerificationCode,
-    verification_code,
-    hasAnyRealAccount,
-    openRealAccountSignup,
-    setResetTradingPasswordModalOpen,
-    toggleAccountSignupModal,
-    toggleResetPasswordModal,
-    setNewEmail,
-    toggleResetEmailModal,
-    toggleUpdateEmailModal,
-}) => {
+const Redirect = observer(() => {
+    const history = useHistory();
+    const { client, ui } = useStore();
```

Comment:
Do we really need to bring the stores into this? We're trying to remove dependencies on the stores, right? 🤔

Reply:
These changes are from an old [PR](https://github.com/binary-com/deriv-app/pull/9778), that PR was closed because of inactivity that's why I created a new PR for this. These are old implementation we just changed the connect method to use useStore hook.